crates/op-rbuilder/src/args/mod.rs

```diff
@@ -70,13 +70,17 @@ impl CliExt for Cli {
 
     /// Returns the type of builder implementation that the node is started with.
     /// Currently supports `Standard` and `Flashblocks` modes.
-    fn builder_mode(&self) -> BuilderMode {
+    fn builder_mode(&self) -> Option<BuilderMode> {
```

Comment:
Can we revert this change? We want to default for now to Standard if nothing else is set.